docs/source/trainer/checkpointing.rst

```diff
@@ -329,6 +329,34 @@ and delete the checkpoints from the local disk. The checkpoints will be located
 ``checkpoints/ep3.pt`` for third epoch's checkpoints, for example. The full URI in this case would be:
 ``s3://my_bucket/checkpoints/ep3.pt``.
 
+For uploading checkpoints to Coreweave's object store, the code is very similar to the
```

Comment:
can you link to some docs about Coreweave's object store?

Reply:
done